app/models/community.rb

```diff
@@ -264,18 +262,8 @@ class Community < ActiveRecord::Base
 
   attr_accessor :terms
 
-  def name(locale=nil)
-    if locale
-      if cc = community_customizations.find_by_locale(locale)
-        cc.name
-      else
-        community_customizations.find_by_locale(locales.first).name
-      end
-    else
-      # TODO: this is not required any more when we remove "name" column,
-      # from community, this should be removed after that.
-      read_attribute(:name)
-    end
+  def name(locale)
+    community_customizations.where(locale: locale).first.name
```

Comment:
In case of missing locale this fails with null pointer. Should decide if it's an error to call with non-existing locale and throw an ArgumentError with good error message or then decide this is acceptable and define a default value.